src/pages/docs/runbooks/runbook-examples/terraform/provision-aws-with-terraform.md

```diff
@@ -278,8 +278,22 @@ workerPool="#{Project.Octopus.Server.WorkerPool}"
 machinePolicy="#{Project.Octopus.Server.MachinePolicy}"
 space="#{Project.Octopus.Server.Space}"
 
-sudo apt-key adv --fetch-keys "https://apt.octopus.com/public.key"
-sudo add-apt-repository "deb https://apt.octopus.com/ stretch main"
+apt-get update && apt-get install --no-install-recommends gnupg curl ca-certificates apt-transport-https && \
+  install -m 0755 -d /etc/apt/keyrings
+
+curl -fsSL https://apt.octopus.com/public.key | sudo gpg --dearmor -o /etc/apt/keyrings/octopus.gpg
+
+chmod a+r /etc/apt/keyrings/octopus.gpg
+
+echo \
+  "deb [arch="$(dpkg --print-architecture)" signed-by=/etc/apt/keyrings/octopus.gpg] https://apt.octopus.com/ \
+  stable main" | \
+  tee /etc/apt/sources.list.d/octopus.list > /dev/null
+
+# for legacy Ubuntu (< 18.04) use
+# apt-key adv --fetch-keys https://apt.octopus.com/public.key
+# add-apt-repository "deb https://apt.octopus.com/ stable main"
+
 sudo apt-get update
 sudo apt-get install tentacle
```

Comment:
We'll want to add the `-y` to line 298 as well.

Reply:
Added.